Automattic/jetpack
# Jetpack Backup: Use Purchases to Display Jetpack Backup States

#### Changes proposed in this Pull Request:

This updates the Plans page to use the `purchases` state to decide what to show for Jetpack Backup.

To accomplish this:
1. `purchases` is added to the Plans component via `getActiveSitePurchases()`
2. The `ProductCard` from Calypso is copied over to Jetpack. (I found it required very little modification to do so, and doing this greatly sped up the UI development.)

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No

#### Testing instructions:
Go to `/wp-admin/admin.php?page=jetpack#/plans` and test the Jetpack Backup card:
1. A free plan should show the upgrade card.
2. Both Jetpack Backup Daily and Realtime should be shown on the card when purchased by themselves.
3. Premium and Professional plans should show Jetpack Daily and Realtime, respectively.

An easy way to test is to use one site, and use the "Manage Subscription" button to remove the plan/product to purchase a new one. Note that in cases of overlap such as a premium plan having also purchased Jetpack Backup Realtime, the highest tier option is picked for display.

#### Proposed changelog entry for your changes:
* None needed, part of Jetpack Backup project.
